app/src/main/java/io/scalac/degree/android/fragment/talk/TalkFragment.java

```diff
@@ -1,254 +1,136 @@
-    int scheduledIconColor;
-
-    @ColorRes(R.color.scheduled_not_star_color)
-    int notscheduledStarColor;
-
-    @Pref
-    Settings_ settings;
-
-    private String talkId;
-
-    @Override
-    public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-    }
+    private boolean isHideToolbarView = false;
 
     @AfterViews
     void afterViews() {
-        setHasOptionsMenu(true);
-        setupViews(slotModel);
+        setupMainLayout();
     }
 
-    public void setupViews(final SlotApiModel slotModel) {
-        talkId = slotModel.talk.id;
-
-        topic.setText(Html.fromHtml(slotModel.talk.title));
-
-        desc.setText(Html.fromHtml(slotModel.talk.summaryAsHtml));
-        desc.setMovementMethod(LinkMovementMethod.getInstance());
-
-        final Context appContext = getActivity().getApplicationContext();
-        final DateFormat dateFormat = android.text.format.DateFormat
-                .getLongDateFormat(appContext);
-        final DateFormat timeFormat = android.text.format.DateFormat
-                .getTimeFormat(appContext);
-
-        date.setText(dateFormat.format(slotModel.fromTimeMillis));
-        start.setText(timeFormat.format(slotModel.fromTimeMillis));
-        end.setText(timeFormat.format(slotModel.toTimeMillis));
-
-        room.setText(slotModel.roomName);
-
-        final List<TalkSpeakerApiModel> speakers = slotModel.talk.speakers;
-        final TalkSpeakerApiModel firstSpeaker = speakers.get(0);
-        speaker.setText(firstSpeaker.name);
-
-        final TalkSpeakerApiModel secondSpeaker = speakers.size() > 1 ? speakers.get(1) : null;
+    @Click(R.id.talkDetailsScheduleBtn)
+    void onScheduleButtonClick() {
+        // TODO Schedule unschedule talk!
+    }
 
-        if (secondSpeaker != null) {
-            speaker2.setText(secondSpeaker.name);
-        } else {
-            speaker2.setVisibility(View.GONE);
-        }
+    @Override
+    public void onOffsetChanged(AppBarLayout appBarLayout, int offset) {
+        int maxScroll = appBarLayout.getTotalScrollRange();
+        float percentage = (float) Math.abs(offset) / (float) maxScroll;
 
-        setupNotificationView();
-
-        scheduleViewContainer.setOnClickListener(v -> {
-            if (notificationsManager.isNotificationScheduled(slotModel.slotId)) {
-                notificationsManager.unscheduleNotification(slotModel.slotId, true);
-                setupNotActiveScheduleView();
-            } else {
-                final NotificationsManager.ScheduleNotificationModel model =
-                        NotificationsManager.ScheduleNotificationModel.create(slotModel, true);
-                notificationsManager.scheduleNotification(model);
-                setupActiveScheduleView();
-            }
-        });
-
-        talkVoter.getVotesCountForTalk(settings.activeConferenceCode().get(), talkId, this);
-    }
+        if (percentage == 1f && isHideToolbarView) {
```

Comment:
`shouldHideToolbarView` w sensie zmienna? i bym sie przyjebal ze 1f to nie `percentage`, wez pomnoz przez 100 i sprawdz czy jest 100 ;)